app/newRoot.css

```diff
@@ -113,18 +115,20 @@ h2 {
 }
 
 .xs {
+  font-size: 14px;
   letter-spacing: -0.14px;
 }
 
 .xs-bold {
+  font-size: 14px;
   letter-spacing: -0.14px;
   font-weight: 600;
 }
 
 /* color classes */
 
 .black {
-  color: var(--colors-black, #000);
+  color: var(--text-black);
```

Comment:
is `--text-black` defined? I can't see it

Reply:
I don't remember adding this line, weird. Fixed now.